Fix issue where response was dropped if the server returns a `String` message id (#66)
dart-lang/json_rpc_2: CHANGELOG.md

## 2.2.3-dev (unreleased)

> * Fix issue where response was ignored when server returned a `String`
>   message id.

Reviewer: ```suggestion
* Accept responses even if the server converts the ID to a String.
```
